pyproject.toml

```diff
@@ -6,6 +6,7 @@ requires = [
     "numpy >=2.0.0, <3",
     "pkgconfig",
     "setuptools >=77",
+    "tempita>=0.6.0"
```

Comment:
Remember to update ci/triage_build.sh as well.

I think we also need a release note on this (for the down-stream packagers).